docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -169,7 +169,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``ewm``                     | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``expanding``               | N                               |                                  |                                                    |
+| ``expanding``               | P                               |                                  | ``N`` if ``axis = 1``                              |
```

Comment:
Can we match this documentation with that of rolling?

Reply:
Good point, done